cmake/Modules/uninstall.cmake

```diff
@@ -0,0 +1,80 @@
+set(MANIFEST "${CMAKE_BINARY_DIR}/install_manifest.txt")
```

Comment:
curios: is this `uninstall.cmake` a custom/internal file, or is it referenced from somewhere else?

Reply:
It's a custom module and it should only be used by the top directory CMakeLists.txt when uninstalling.